test/functional/api/vim_spec.lua

```diff
@@ -115,6 +115,29 @@ describe('API', function()
       nvim('exec','autocmd BufAdd * :let x1 = "Hello"', false)
       nvim('command', 'new foo')
       eq('Hello', request('nvim_eval', 'g:x1'))
+
+      -- Script scope (s:)
+      eq('ahoy! script-scoped varrrrr', nvim('exec', [[
+          let s:pirate = 'script-scoped varrrrr'
+          func! s:avast_ye_hades(s) abort
+            return a:s.' '.s:pirate
+          endf
+          echo <sid>avast_ye_hades('ahoy!')
+        ]], true))
+
+      eq('script-scoped varrrrr', nvim('exec', [[
```

Comment:
This case is currently the same as last one. What I mean is you want to make it `func! Avast_ye_hades()`. and then call it in a second `nvim_eval` api request from the lua client.

Reply:
I have the added the test case.